tests/test_browser.py

```diff
@@ -3225,6 +3225,15 @@ def test_async_virtual_2(self):
       print(opts)
       self.btest('browser/async_virtual_2.cpp', '1', args=['-O' + str(opts), '-s', 'ASSERTIONS=1', '-s', 'SAFE_HEAP=1', '-profiling'] + self.get_async_args())
 
+  # Test async sleeps in the presence of invoke_* calls, which can happen with
+  # longjmp or exceptions.
+  @parameterized({
+    '0': ([],), # noqa
```

Comment:
Can you call these `O0` and `O2`? I find just _1 and _3 a little less meaningful as test suffixes.